doc/changelog/06-Ltac2-language/16655-body-hyp-match.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  :tacn:`match! goal` support for matching hypothesis bodies
```

Comment:
I think it's worth mentioning which Ltac2 definitions have different types, as this might cause incompatibilities

Reply:
changed